src/MR.AspNetCore.Jobs.PostgreSQL/Server/ExpirationManager.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using MR.AspNetCore.Jobs.Models;
+
+namespace MR.AspNetCore.Jobs.Server
+{
+	public class PostgreSQLExpirationManager : EFCoreExpirationManager<JobsDbContext>
+	{
+		public PostgreSQLExpirationManager(
+			ILogger<PostgreSQLExpirationManager> logger,
+			PostgreSQLOptions options,
+			IServiceProvider provider)
+			: base(logger, options, provider)
+		{
+		}
+
+		protected override string CreateDeleteTopQuery(string schema, string table)
+		{
+			return $@"
+DELETE
+FROM ""{schema}"".""{table}""
+WHERE ExpiresAt < @now
```

Comment:
postgres didn't like this field syntax. seems to need everything fully qualified with schema. this worked for me:

```csharp
return $@"
DELETE
FROM ""{schema}"".""{table}""
WHERE ""{schema}"".""{table}"".""ExpiresAt"" < @now
AND ctid IN (
    SELECT ctid
    FROM  ""{schema}"".""{table}""
    ORDER BY ""{schema}"".""{table}"".""ExpiresAt""
    LIMIT @count
)";
```

Reply:
Thanks. I have little experience with postgres, and not enough time to test things out (which is why this PR is still pending).

Reply:
With little simplification, I'd guess this should also work:
```
return $@"
DELETE
FROM ""{schema}"".""{table}""
WHERE ""ExpiresAt"" < @now
AND ctid IN (
    SELECT ctid
    FROM  ""{schema}"".""{table}""
    ORDER BY ""ExpiresAt""
    LIMIT @count
)";
```

The problem is with the LIMIT not being acceptable with using CTID, I think.